pom.xml

```diff
@@ -30,7 +48,7 @@
             <groupId>org.assertj</groupId>
             <artifactId>assertj-core</artifactId>
             <version>3.18.1</version>
-            <scope>test</scope>
+            <scope>com.example.project.test</scope>
```

Comment:
this does not seem right, please refer to https://www.baeldung.com/maven-dependency-scopes for more details